recipes-kernel/linux/files/bcm2712-raspberrypi5-xen.dtso

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024 EPAM systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+*/
+/dts-v1/;
+/plugin/;
+
+&chosen {
+	#address-cells = <0x2>;
+	#size-cells = <0x2>;
+	xen,xen-bootargs = "console=dtuart dtuart=/soc/serial@7d001000 dom0_mem=128M dom0_max_vcpus=1 xsm=flask flask=permissive";
+	xen,dom0-bootargs = "console=hvc0 earlycon=xen earlyprintk=xen clk_ignore_unused root=/dev/ram0";
```

Comment:
could you drop bootargs here pls - they set in [xt-rpi-u-boot-scr.bb](https://github.com/xen-troops/meta-xt-rpi5/commit/bcfe5ce4107a178c6e435684f615fca071fb0df6#diff-bf65ca7b48fa2b6b3ea083f52969ca7e1ec6514ce23dd7c245ccc95774cec9cb)